### PR TITLE
Create 0xTrade

### PR DESCRIPTION
**Pull request by reeveshen**
add new chain  0xTrade

### CODE REVIEW

**Comment by MoGenius5420** [comment]
`gh pr checkout 1644`

File: _data/chains/0xTrade [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "0XTade",
+  "chain": "0XTade Chain",
+  "rpc": ["https://node.0xtchain.com"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "0XT"
```
Comment (ligi) [inline _data/chains/0xTrade:7]:
```suggestion
    "name": "0XT",
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/0xTrade [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "nativeCurrency": {
+    "name": "0XT",
+    "symbol": "0XT",
+    "decimals": 18
+  },
+  "infoURL": "https://www.0xtrade.finance/",
+  "shortName": "0xt
+  "chainId": 10248
+  "networkId": 10248
+  "icon": "0xtrade
```
Comment (ligi) [inline _data/chains/0xTrade]:
```suggestion
  "shortName": "0xt",
  "chainId": 10248,
  "networkId": 10248,
  "icon": "0xtrade",
```

File: _data/chains/eip155-10248.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "nativeCurrency": {
+    "name": "0XT",
+    "symbol": "0XT",
+    "decimals": 18
+  },
+  "infoURL": "https://www.0xtrade.finance/",
+  "shortName": "0xt",
+  "chainId": 10248,
+  "networkId": 10248,
+  "icon": "0xtrade",
```
Comment (ligi) [inline _data/chains/eip155-10248.json:15]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Pull request merged**